pkg/patterns/declarative/pkg/applier/direct.go

```diff
@@ -53,6 +69,7 @@ func (d *DirectApplier) Apply(ctx context.Context,
 	applyOpts.DeleteOptions = &cmdDelete.DeleteOptions{
 		IOStreams: ioStreams,
 	}
+	applyOpts.DeleteOptions.Filenames = []string{tmpFile.Name()}
```

Comment:
Looking at the code, it seems that despite being in DeleteOptions.Filenames, this the filename of the objects we want to apply.  Is that right?  If that's right, do you mind adding a comment?